packages/app/context/theme.tsx

```diff
@@ -59,6 +61,28 @@ const ThemeContext = createContext({
 export const ThemeProvider = ({ children }) => {
   const [state, dispatch] = useReducer(reducer, initialState);
 
+  useEffect(() => {
+    const fetchTheme = async () => {
+      let storedIsEnabled;
+      if (Platform.OS === 'web') {
```

Comment:
pretty sure this will crash on native, but we have an abstracted hook for accessing storage on either native/web that we could use here

Reply:
Ok, I did change it to use the hook now. check it please